web/tendrl/cluster_import.rst

```diff
@@ -31,10 +32,18 @@ Test Steps
 
 .. test_action::
    :step:
-      Follow wizard for importing cluster.
+      | Follow wizard for importing cluster.
+      | Volume profiling (checkbox ``Enable Volume profiling.``) leave enabled (should be the default).
    :result:
       Cluster is successfully imported.
 
+.. test_action::
+   :step:
+      | Login to one of the gluster storage server via SSH and check if Volume Profiling is enabled.
+      | ``gluster volume profile <VOLUME-NAME> info``
```

Comment:
There has to be some volume present, for this step. Probably mention that in set-up for this test.
Moreover global volume profiling enabled should force the volume profiling for all existing volumes and also for new ones. So there should be also step where a new volume is created and volume profiling is checked for this volume.

Reply:
> There has to be some volume present, for this step. Probably mention that in set-up for this test.
This is good point, I'm just thinking if it is necessary, to mention it explicitly here, because volume is now created during default setup.

> Moreover global volume profiling enabled should force the volume profiling for all existing volumes and also for new ones. So there should be also step where a new volume is created and volume profiling is checked for this volume.
This is out of scope for this test.

Reply:
You're right the volume is created by default in our automated tests, but this test should not depend on set-up of our automated tests. And if it counts with such set-up it should be mentioned somewhere, not necessary in the test itself though.